packages/fxa-settings/src/lib/config.ts

```diff
@@ -1,17 +1,103 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { deepMerge } from './utilities';
+
+export const META_CONFIG = 'fxa-config';
+
 export interface Config {
   env: string;
+  version: string;
   sentry: {
     url: string;
     dsn: string;
   };
-  version: string | undefined;
+  servers: {
+    gql: {
+      url: string;
+    };
+    auth: {
+      url: string;
```

Comment:
👍 We probably want to note somewhere (not here) what we need to use the auth server for in case someone else picks up those issues.